docs/source/conf.py

```diff
@@ -3,15 +3,15 @@
 
 import os
 
-# -- General configuration -----------------------------------------------------
+# Add any Sphinx extension module names, as strings. They can be extensions
```

Comment:
@AhmetCanSolak may know more, but I think this file was auto-generated.

Reply:
Some exceeded the 79-character limit, so it automatically edited some things. I made some changes that it could not fix automatically.